### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ZakMos**
update to new version: AlmaLinux release 8.4 (Electric Cheetah)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,11 +24,11 @@ In order to convert your EL8 operating system to AlmaLinux do the following:
    ```shell
    # check release file
    $ cat /etc/redhat-release 
-   AlmaLinux release 8.3 (Purple Manul)
+   AlmaLinux release 8.4 (Purple Manul)
```
Comment (andrewlukoshko) [inline README.md:27]: Please change codename to Electric Cheetah

**Review by andrewlukoshko — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ZakMos) [reply README.md:27]:
```suggestion
   AlmaLinux release 8.4 (Electric Cheetah)
```

**Review by ZakMos — COMMENTED** [review]
done!

File: README.md [hunk 2]
```diff
@@ -24,11 +24,11 @@ In order to convert your EL8 operating system to AlmaLinux do the following:
    ```shell
    # check release file
    $ cat /etc/redhat-release 
-   AlmaLinux release 8.3 (Purple Manul)
+   AlmaLinux release 8.4 (Electric Cheetah)
    
    # check that the system boots AlmaLinux kernel by default
    $ sudo grubby --info DEFAULT | grep AlmaLinux
-   title="AlmaLinux (4.18.0-240.el8.x86_64) 8"
+   title="AlmaLinux (4.18.0-305.1.el8.x86_64) 8.4"
```
Comment (andrewlukoshko) [inline README.md:31]:
```suggestion
   title="AlmaLinux (4.18.0-305.el8.x86_64) 8.4"
```

**Review by andrewlukoshko — CHANGES_REQUESTED** [review]

**Pull request merged**